debian/matrix-synapse-py3.triggers

```diff
@@ -0,0 +1,9 @@
+# Register interest in Python interpreter changes (Python 2 for now); and
+# don't make the Python package dependent on the virtualenv package
+# processing (noawait)
+interest-noawait /usr/bin/python3.5
+interest-noawait /usr/bin/python3.6
+interest-noawait /usr/bin/python3.7
+
+# Also provide a symbolic trigger for all dh-virtualenv packages
```

Comment:
s/provide/listen to/?

Reply:
well, let's not just cargo-cult stuff without understanding it please...

Reply:
well i'm also cargo culting it from the dh_virtualenv docs, I have no idea what it means in a real sense.

Reply:
Sigh; I see what you mean. Having stared at the docs, the dh-virtualenv source, the generated postinst... I also have no idea what they are trying to achieve with this.

Reply:
debiaaaaaaaaaaaaaaaaaaaaaaaaaaaaaan :D